diff -up yudit-3.1.0/yudit.properties.in.1~ yudit-3.1.0/yudit.properties.in
--- yudit-3.1.0/yudit.properties.in.1~	2023-02-09 05:32:11.000000000 +0100
+++ yudit-3.1.0/yudit.properties.in	2023-02-14 21:07:33.674329408 +0100
@@ -43,7 +43,37 @@ yudit.datapath=
 # 2. this comma,separated,path, like /my/dir1,/my/dir2
 # 3. @YUDIT_DATA@/fonts
 # Please use /somedir/** to extend search to subdirectories (till level5)
-yudit.fontpath=~/.local/share/fonts/**,~/Library/Fonts/**,/usr/share/fonts/**,/System/Library/Fonts/**
+yudit.fontpath=~/.local/share/fonts/**,/usr/share/fonts/**,\
+/usr/X11R6/lib/X11/fonts/drakfont,\
+/usr/X11R6/lib/X11/fonts/drakfont/ttf,\
+/usr/share/fonts/ttf/arabic,\
+/usr/share/fonts/ttf/armenian,\
+/usr/share/fonts/ttf/bengali,\
+/usr/share/fonts/ttf/big5,\
+/usr/share/fonts/ttf/cyrillic,\
+/usr/share/fonts/ttf/decoratives,\
+/usr/share/fonts/ttf/ethiopic,\
+/usr/share/fonts/ttf/gb2312,\
+/usr/share/fonts/ttf/georgian,\
+/usr/share/fonts/ttf/hebrew,\
+/usr/share/fonts/ttf/japanese,\
+/usr/share/fonts/ttf/kannada,\
+/usr/share/fonts/ttf/korean,\
+/usr/share/fonts/ttf/tamil,\
+/usr/share/fonts/ttf/thai,\
+/usr/share/fonts/ttf/tscii,\
+/usr/share/fonts/ttf/vera,\
+/usr/share/fonts/ttf/viet,\
+/usr/share/fonts/ttf/western,\
+/usr/share/fonts/otf/mdk,\
+/opt/ttfonts,\
+/opt/ttfonts/arabic,\
+/opt/ttfonts/chinese,\
+/opt/ttfonts/ethiopic,\
+/opt/ttfonts/georgian,\
+/opt/ttfonts/japanese,\
+/opt/ttfonts/msfonts,\
+/usr/X11R6/lib/X11/fonts/TTF
 
 #
 # This is where the .syn (Syntax Highlighting) files are
@@ -118,7 +148,7 @@ yudit.freehand.converter=kanji
 # If you specify yudit.default.language=default, then LANG environment 
 #    variable is used. Be careful - you may not have the font!
 #
-yudit.default.language=
+yudit.default.language=default
 
 #
 # Things will be put on X11 clipboard with this encoding.
@@ -243,8 +273,8 @@ yudit.editor.fonts=default,TrueType,Seri
 # fcitx only supports preedit-under-status-under input style
 #
 # On Mac F11 is blocked
-yudit.editor.inputs=straight,unicode,@YUDIT_F3@,Hungarian,freehand,Hangul,\
-Emoji,Mnemonic,Esperanto,OldHungarian,Kana,HungarianRunes
+yudit.editor.inputs=x-utf-8:en_US.UTF-8,straight,uim-anthy,unicode,kinput2,@YUDIT_F3@,Hungarian,freehand,Hangul,\
+Emoji,Mnemonic,Esperanto,x-utf-8:en_GB,OldHungarian,x-ibus:en_US.utf-8,Kana,HungarianRunes
 
 #
 # kinput2:ja_JP, for instance will appear as x-kinput2 to save room
@@ -259,7 +289,7 @@ Emoji,Mnemonic,Esperanto,OldHungarian,Ka
 #
 # On Ubuntu fcitx only supports preedit-under-status-under input style.
 #
-yudit.editor.xinputs=fcitx:en_US.utf-8,ibus:en_US.utf-8,SCIM:en_US.utf-8,interxim,utf-8:en_GB,utf-8:hu_HU,kinput2:ja_JP.eucJP,xcin:zh_TW.big5,Ami:ko_KR,xcin-zh_CN.GB2312:zh_CN.GB2312
+yudit.editor.xinputs=utf-8:en_US.UTF-8,fcitx:en_US.utf-8,ibus:en_US.utf-8,SCIM:en_US.utf-8,interxim,utf-8:en_GB,utf-8:hu_HU,kinput2:ja_JP.eucJP,xcin:zh_TW.big5,Ami:ko_KR,xcin-zh_CN.GB2312:zh_CN.GB2312
 
 #
 # The x input style for the editor window.
@@ -272,9 +302,9 @@ yudit.editor.xinputs=fcitx:en_US.utf-8,i
 # On Ubuntu fcitx only supports preedit-under-status-under input style.
 yudit.editor.xinputs.style=preedit-under-status-under
 
-yudit.editor.fontsize=18
-yudit.editor.font=default
-yudit.editor.input=straight
+yudit.editor.fontsize=16
+yudit.editor.font=TrueType
+yudit.editor.input=x-utf-8:en_US.UTF-8
 
 yudit.fonts=default,TrueType,Bitmap,Japanese,Courier,Times,Helvetica,Lucida,Misc
 
@@ -312,7 +342,16 @@ yudit.fonts=default,TrueType,Bitmap,Japa
 #    If not specified it is used as LR or RL and for RL mirrored glyphs 
 #    it will get mirrored by software.
 #
-yudit.font.TrueType=Arial.ttf,ZenKakuGothicNew-Regular.ttf,gulim.ttf,ogulim.ttf:mslvt,raghu.ttf:deva,tsckanna.ttf:tscii,dc-font.ttf:mlym,MalOtf.ttf:mlym,code2000.ttf,code2001.ttf:unicode:RL,cyberbit.ttf,OldHungarian_Full.ttf:unicode:RL,yudit.ttf,TwitterColorEmoji-SVGinOT.ttf:emoji,EmojiOne_Color.otf:emoji
+yudit.font.TrueType=Arial.ttf,ZenKakuGothicNew-Regular.ttf,gulim.ttf,ogulim.ttf:mslvt,raghu.ttf:deva,tsckanna.ttf:tscii,dc-font.ttf:mlym,MalOtf.ttf:mlym,code2000.ttf,code2001.ttf:unicode:RL,cyberbit.ttf,OldHungarian_Full.ttf:unicode:RL,yudit.ttf,TwitterColorEmoji-SVGinOT.ttf:emoji,EmojiOne_Color.otf:emoji,msgothic.ttf,kochi-gothic.ttf,gulim.ttf,\
+caslr.ttf,fogcaslr.ttf,l_10646.ttf,tahoma.ttf,\
+bpgutf8m.ttf,gfzemenu.ttf,artsnk_b.ttf,bsmi00lp.ttf,gbsn00lp.ttf,\
+msmincho.ttf,norasi_n.ttf,Pigiarniq_Regular.ttf,urdunast.ttf,ballrn__.ttf,\
+mssong.ttf,\
+Sampige.ttf:knda,MuktiRegular.ttf:beng,\
+ogulim.ttf:mslvt,raghu.ttf,tsckanna.ttf:tscii,dc-font.ttf:mlym,MalOtf.ttf:mlym,\
+code2000.ttf,code2001.ttf:unicode:RL,\
+cyberbit.ttf,rovstd.ttf:rovas:LR,yudit.ttf,\
+Kacst-Qr.ttf
 
 #
 # For convention always have one yudit.font.Bitmap
